ui/round/src/ctrl.ts

```diff
@@ -701,14 +701,14 @@ export default class RoundController {
           return msg;
         });
 
-        if (!this.nvui) {
+        if (!this.nvui && d.pref.submitMove) {
```

Comment:
This doesn't mean the current is has move confirmation

Reply:
It doesn’t?  How do you check that then?

Escape doesn’t work in the command bar during games because of this bind